app/components/post_header/index.js

```diff
@@ -36,7 +37,8 @@ function makeMapStateToProps() {
             isPendingOrFailedPost: isPostPendingOrFailed(post),
             isSystemMessage: isSystemMessage(post),
             overrideUsername: post.props && post.props.override_username,
-            theme: getTheme(state)
+            theme: getTheme(state),
+            isFlagged: isPostFlagged(post.id, myPreferences)
```

Comment:
I wonder if we should get this in `post` and pass it as an ownProp to post_header and post_body as with this change we are currently looking in both places if the post is flagged

Reply:
@enahum Yes that makes sense. I'll make the change.